extensions/panache/mongodb-panache-common/deployment/src/main/java/io/quarkus/mongodb/panache/deployment/BasePanacheMongoResourceProcessor.java

```diff
@@ -87,7 +94,8 @@ public void buildReactive(CombinedIndexBuildItem index,
         List<PanacheMethodCustomizer> methodCustomizers = methodCustomizersBuildItems.stream()
                 .map(bi -> bi.getMethodCustomizer()).collect(Collectors.toList());
 
-        processTypes(index, transformers, reflectiveClass, propertyMappingClass, getReactiveTypeBundle(),
+        processTypes(index, transformers, reflectiveClass, propertyMappingClass,
```

Comment:
I don't understand how come we're getting formatting here, isn't that auto-formatted?

Reply:
it is.  there used to be an extra parameter there before I realized it wasn't necessary which probably cause the wrap there.

Reply:
It's still weird, I thought formatting was deterministic. But whatever.

Reply:
I had edited the source and added a parameter causing the new wrap.  then removed it when it turned up unnecessary (the mongodb case is simpler than hibernate's).  But i'll clean that up so the diff is smaller.